test/e2e/kubelet_perf.go

```diff
@@ -242,7 +247,7 @@ var _ = Describe("Kubelet [Serial] [Slow]", func() {
 			itArg := testArg
 			podsPerNode := itArg.podsPerNode
 			name := fmt.Sprintf(
-				"for %d pods per node over %v", podsPerNode, monitoringTime)
+				"[Feature:NodePerformance] resource tracking for %d pods per node", podsPerNode)
```

Comment:
My comment was valid only for ">35" case. For "normal" ones you shouldn't add a Feature tag, as this will disable the test.